easybuild/easyconfigs/g/genomepy/genomepy-0.15.0-foss-2022a.eb

```diff
@@ -0,0 +1,72 @@
+easyblock = 'PythonBundle'
+
+name = 'genomepy'
+version = '0.15.0'
+
+homepage = 'https://github.com/vanheeringen-lab/genomepy'
+description = "genomepy is designed to provide a simple and straightforward way to download and use genomic data"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('tqdm', '4.64.0'),
+    ('Biopython', '1.79'),
+    ('mygene', '3.2.2'),
+    ('pyfaidx', '0.7.1'),
+    ('PyYAML', '6.0'),
+    ('protobuf-python', '3.19.4'),
+]
+
+use_pip = True
+
+# avoid hatchling requirement to install infercnvpy
```

Comment:
`infercnvpy`? I think this comment may have been copied from another EasyConfig, you probably forgot to change the name here :)

Reply:
Ok, fixed it myself since it was such a minor change...